### PR TITLE
Logs: Add custom fields

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-2212.

### CODE REVIEW

File: content/cloudflare-one/analytics/logs/audit-logs.md [hunk 1]
```diff
@@ -36,17 +36,17 @@ Access authentication logs contain the following fields:
...
+| **action** | The event that occurred, such as a login attempt. |
 | **allowed** | The result of the authentication event. |
 | **created\_at** | The event timestamp. |
 | **connection**  | The IdP used to authenticate. |
-| **country** | The country associated with the user’s IP address |
-| **ray\_id** | A unique identifier for every request through Cloudflare |
-| **app\_type** | Specifies if the app is self-hosted or SaaS |
+| **country** | The country associated with the user’s IP address. |
+| **ray\_id** | A unique identifier for every request through Cloudflare. |
+| **app\_type** | Specifies if the app is self-hosted or SaaS. |
```
Comment (angelampcosta) [inline content/cloudflare-one/analytics/logs/audit-logs.md]:
```suggestion
| **app\_type** | The type specifies if the app is self-hosted or SaaS. |
```

Comment (angelampcosta) [reply content/cloudflare-one/analytics/logs/audit-logs.md]: This was the only one that started by a verb.

File: content/cloudflare-one/analytics/logs/audit-logs.md [hunk 2]
```diff
@@ -137,21 +137,23 @@ Cloudflare Logpush can be used to gather and send specific request headers from
...
+
+In addition to the HTTP request fields available in Cloudflare Enterprise logging, requests made to applications behind Access include the `cf-access-user` field, which contains the user identity string. This offers another tool for auditing user behavior. To add the `cf-access-user` field to your HTTP request logs, you must add it as a custom field. Refer to [Configure custom fields](/logs/reference/logpush-api-configuration/custom-fields/) for instructions.
+
+Keep in mind that Access does not log all interactions. For example, per-request audit logs can indicate that a specific user visited `domain.com/admin` and then `domain.com/admin/panel`, but the logs can only identify user interactions that result in a new HTTP request.
 
 ### Cloudflare logpush integration
 
 Access integrates with the Cloudflare Logpush API, so you can export per-request audit logs to third-party Security Information and Event Management (SIEM) tools.
 
 Cloudflare Logpush pushes Enterprise customers' HTTP request logs, including Access user identity, to a cloud storage provider every 5 minutes.
```
Comment (angelampcosta) [inline content/cloudflare-one/analytics/logs/audit-logs.md]:
```suggestion
Cloudflare Logpush pushes Enterprise customers' HTTP request logs, including Access user identity, to a cloud storage provider every five minutes.
```

File: content/logs/reference/log-fields/_index.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ weight: 51
 The datasets below describe the fields available by log category. The list of fields is also accessible directly from the API:
 `https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/datasets/<DATASET>/fields` for zone-scoped datasets or `https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/logpush/datasets/<DATASET>/fields` for account-scoped datasets, where the **dataset** argument indicates the log category (such as `http_requests`, `spectrum_events`, `firewall_events`, `nel_reports`, or `dns_logs`).
 
-Zone-scoped HTTP requests are available in both Logpush and Logpull. All other datasets are only available through Logpush.
+Zone-scoped HTTP requests are available in both Logpush and Logpull, except for [custom fields](/logs/reference/logpush-api-configuration/custom-fields/), which are only available in Logpush. All other datasets are only available through Logpush.
```
Comment (angelampcosta) [inline content/logs/reference/log-fields/_index.md:12]: Line 22 Teams Logs sounds strange, maybe just Logs.

**Review by angelampcosta — APPROVED** [review]

File: content/logs/reference/log-fields/_index.md (hunk 3 above)
Comment (pedrosousa) [reply content/logs/reference/log-fields/_index.md:12]: I'll change it to "Zero Trust logs", since that's the updated page name.

**Pull request merged**